### PR TITLE
DEVOP-955: Add management scripts

### PR DESCRIPTION
**Pull request by sboardwell**
- Install ansible, Makefile, tweaks
- Add test files
- Add DEBUG=1 option
- Add datadog hdfs configs
- Add spotlight configuration
- Add service, env, version tags to spotlight
- Add basic datadog config as fallback
- Add option to check EMR buckets for config directly
- Add jmx integration changes for hdfs information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,139 @@
...
+```bash
+$ make help
+
+
+Usage:
+  make <target>
+
+Targets:
+  dd-checks            For <TAG> (default: latest), list all config examples found in "/etc/datadog-agent/conf.d"
+  dd-check/%           For <TAG> (default: latest), output the "/etc/datadog-agent/conf.d/<%>.d/conf.yaml.example
```
Comment (willbenica) [inline README.md]: missing `"` at the end of the line

File: scripts/functions.sh [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    $SUDO_CMD apt-add-repository --yes --update ppa:ansible/ansible
+    $SUDO_CMD apt install -y ansible
+  else
+    printf "\033[31mYour OS or distribution are not supported by this script.\033[0m\n"
+    exit;
+  fi
+}
+
+curl_ec2() {
+  local ec2MetaUrl='http://169.254.169.254/latest'
```
Comment (willbenica) [inline scripts/functions.sh:101]: Is this correct, pointing at a hardcoded URL?

File: tools/datadog/components/basic/basic.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+# default stuff?
```
Comment (willbenica) [inline tools/datadog/components/basic/basic.sh:1]: Coolest bash script ever :-)

**Review by willbenica — COMMENTED** [review]
Found a missing `"` in the README, and I saw that we are hard coding a URL in the `functions.sh` -> `curl_ec2()` function.

File: scripts/functions.sh (hunk 2 above)
Comment (sboardwell) [reply scripts/functions.sh:101]: yes, see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

File: README.md (hunk 1 above)
Comment (sboardwell) [reply README.md]: Fixed with 9e2ace3

**Review by willbenica — COMMENTED** [review]
Looks good to me.

**Review by shishir-narayan — APPROVED** [review]
lgtm

File: tools/datadog/components/emr/task-node.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+init_instance_tags
+CLUSTER_ID=$(get_instance_tag 'aws:elasticmapreduce:job-flow-id')
+CLUSTER_NAME=$(aws emr describe-cluster --cluster-id $CLUSTER_ID --region $AWS_REGION --query 'Cluster.Name' --output text)
+DD_CLUSTER_NAME="${CLUSTER_NAME}_${CLUSTER_ID}"
+HDFS_DATANODE_JMX_URI="http://$(hostname -f):9864"
```
Comment (sboardwell) [inline tools/datadog/components/emr/task-node.sh:5]: Remove

**Pull request merged**